packages/client-core/src/admin/components/settings/tabs/metabase.tsx

```diff
@@ -76,26 +87,25 @@ const MetabaseTab = forwardRef(({ open }: { open: boolean }, ref: React.MutableR
       crashDashboardId: crashDashboardId.value
     }
 
-    const operation = !id.value
-      ? metabaseSettingMutation.create(setting)
-      : metabaseSettingMutation.patch(id.value, setting)
-    operation
-      .then(() => {
-        state.set({ loading: false, errorMessage: '' })
-      })
-      .catch((e) => {
-        state.set({ loading: false, errorMessage: e.message })
-      })
+    // const operation = !id.value
+    //   ? metabaseSettingMutation.create(setting)
+    //   : metabaseSettingMutation.patch(id.value, setting)
+    // operation
+    //   .then(() => {
+    //     state.set({ loading: false, errorMessage: '' })
+    //   })
+    //   .catch((e) => {
+    //     state.set({ loading: false, errorMessage: e.message })
+    //   })
   }
```

Comment:
I think this is commented out by mistake?

Reply:
yes , i'll fix this

Reply:
Done.